src/content/docs/crowdin/project-management/sources/version-management.mdx

```diff
@@ -66,37 +159,74 @@ To set a priority for your branches, follow these steps:
 
 <Steps>
   1. Open your project and go to **Sources > Files**.
-  2. Click an arrow icon next to the needed branch to set the preferred priority.
+  2. Click the arrow icon next to the needed branch to set the preferred priority.
 </Steps>
 
 <Aside>
-  Branch prioritization is only available for [file-based](/creating-project/#project-types) projects.
+  Branch prioritization is only available for [file-based](/creating-project/#file-based-project) projects.
 </Aside>
 
+### Other Actions from the Context Menu
```

Comment:
The rename option could be here. It's a simple and obvious action, no need to describe it with steps